packages/nova-lib/lib/modules/apollo.js

```diff
@@ -1,60 +1,83 @@
-import ApolloClient, { createNetworkInterface } from 'apollo-client';
+import ApolloClient, { createNetworkInterface, createBatchingNetworkInterface } from 'apollo-client';
 import 'isomorphic-fetch';
+
+import { Meteor } from 'meteor/meteor';
+
 import { getSetting } from './settings.js';
 
 const defaultNetworkInterfaceConfig = {
-  path: '/graphql',
-  options: {},
+  path: '/graphql', // default graphql server endpoint
+  opts: {}, // additional fetch options like `credentials` or `headers`
+  useMeteorAccounts: true, // if true, send an eventual Meteor login token to identify the current user with every request
+  batchingInterface: false, // use a BatchingNetworkInterface by default instead of a NetworkInterface
+  batchInterval: 10, // default batch interval
 };
 
 const createMeteorNetworkInterface = (givenConfig = {}) => {
-  const config = _.extend(defaultNetworkInterfaceConfig, givenConfig);
+  const config = { ...defaultNetworkInterfaceConfig, ...givenConfig };
 
   // absoluteUrl adds a '/', so let's remove it first
   let path = config.path;
   if (path[0] === '/') {
     path = path.slice(1);
   }
 
-  // add the rootUrl option, in case tests are made on development server
-  // see https://github.com/TelescopeJS/Telescope/issues/1554#issuecomment-277445915
   const uri = Meteor.absoluteUrl(
-    path, 
-    { rootUrl: getSetting('developmentServerIp', Meteor.absoluteUrl()) }
+    path,
+    { rootUrl: getSetting('developmentServerIp', Meteor.absoluteUrl()) },
   );
-  const networkInterface = createNetworkInterface({
+
+  // allow the use of a batching network interface; if the options.batchingInterface is not specified, fallback to the standard network interface
+  const interfaceToUse = config.batchingInterface ? createBatchingNetworkInterface : createNetworkInterface;
```

Comment:
Sweet the alignment with `apollo` atmosphere package 👍 

Now we need to configure `dataloader` to batch queries server-side too 😄